lib/files_test.go

```diff
@@ -250,21 +250,21 @@ func TestReadLines(t *testing.T) {
 	if errCurr != nil {
 		log.Fatal(errCurr)
 	}
-	installLocation := usr.HomeDir + installPath
+	installLocation := filepath.Join(usr.HomeDir, installPath)
```

Comment:
Using `filepath.Join()`  is probably a better solution. Thanks!

Reply:
just to confirm this line is ok 😄 , it's using `filepath.Join`